Jenkinsfile

```diff
@@ -0,0 +1,67 @@
+import groovy.json.JsonOutput
+
+
+timestamps {
+  node('regular-memory-node') {
+    stage('checkout') {
+      scmInfo = checkout scm
+      println("${scmInfo}")
+      env.GIT_BRANCH = scmInfo.GIT_BRANCH
+      env.GIT_COMMIT = scmInfo.GIT_COMMIT
+    }
+
+    stage('Build') {
+      withCredentials([
+        usernamePassword(credentialsId: '063fc85b-62a6-4181-9d72-873b43488411', usernameVariable: 'AWS_ACCESS_KEY_ID', passwordVariable: 'AWS_SECRET_ACCESS_KEY'),
+        string(credentialsId: 'a791118f-a1ea-46cd-b876-56da1b9bc71c',variable: 'NEXUS_PASSWORD')
```

Comment:
are we still going to use nexus?

Reply:
it is used for login to docker registry where we have image with net6